content/ngf/how-to/data-plane-configuration.md

```diff
@@ -375,6 +373,36 @@ spec:
 EOF
 ```
 
-For the full configuration API, see the `NginxProxy spec` in the [API reference]({{< ref "/ngf/reference/api.md" >}}).
-
 {{< note >}} When sending curl requests to a server expecting proxy information, use the flag `--haproxy-protocol` to avoid broken header errors. {{< /note >}}
+
+---
+
+## Configure infrastructure-related settings
+
+You can configure deployment and service settings for all data plane instances by editing the `NginxProxy` resource at the GatewayClass level. These settings can also be specified under `nginx.` in the Helm values file. 
```

Comment:
I would enhance this to say that you can also set these fields per-Gateway, because that's going to be an important way to do it if a user has multiple Gateways.